helm/hpcc/templates/_warnings.tpl

```diff
@@ -86,7 +86,7 @@ Pass in dict with root and warnings
  {{- end -}}
  {{- range $cname, $ctypes := $ctx.components -}}
   {{- range $id, $component := $ctypes -}}
-   {{- if not $component.disabled -}}
+   {{- if and (ne ($id|toString) "disabled") (not $component.disabled) -}}
```

Comment:
alternatively, instead of explicitly checking "disabled" could check type is not a boolean with (typeIs "bool" $component)